airflow/docker-compose.yaml

```diff
@@ -77,6 +77,7 @@ x-airflow-common:
     - ${AIRFLOW_PROJ_DIR:-.}/logs:/opt/airflow/logs
     - ${AIRFLOW_PROJ_DIR:-.}/config:/opt/airflow/config
     - ${AIRFLOW_PROJ_DIR:-.}/plugins:/opt/airflow/plugins
+    - /Users/andriifeshchenko/Workspace/data_engineering/data-engineering-practice/lesson_02/data:/mnt/data
```

Comment:
should not hardcode local path and store on git

Reply:
fixed with env variable